CHANGELOG.md

```diff
@@ -20,6 +20,7 @@
   - [KAFKA-154](https://jira.mongodb.org/browse/KAFKA-154) Improve the handling and error messaging for Json array config values.
   - [KAFKA-78](https://jira.mongodb.org/browse/KAFKA-78) Added dead letter queue support for the source connector.
   - [KAFKA-157](https://jira.mongodb.org/browse/KAFKA-157) Improved error message for business key errors.
+  - [KAFKA-105](https://jira.mongodb.org/browse/KAFKA-105) Improve `errors.tolerance=all` support in the sink and source connectors.
```

Comment:
`errors.tolerance=all` is a KAFKA connect setting which we've copied.  It allows errors to be ignored outside of the connector (eg. during conversion).  This PR adds support inside the connector where possible, config errors are still errors but message based errors during `Source.poll` & `Sink.put` now honor this setting.